src/main/java/seedu/address/storage/XmlAdaptedPerson.java

```diff
@@ -122,6 +129,11 @@ public Person toModelType() throws IllegalValueException {
         adaptedPerson.setDateBorrow(dateBorrow);
         adaptedPerson.setDateRepaid(dateRepaid);
         adaptedPerson.setLastAccruedDate(lastAccruedDate);
+        if (this.hasOverdueDebt.equals("true")) {
+            adaptedPerson.setHasOverdueDebt(true);
+        } else {
+            adaptedPerson.setHasOverdueDebt(false);
+        }
```

Comment:
I believe you are saving the data in the xml file so that data does not get erased when you reopen it?